h/assets.yaml

```diff
@@ -100,10 +100,14 @@ config:
   output: scripts/config.min.js
   filters: uglifyjs
   contents:
-    - output: scripts/config.js
+    - output: scripts/config.accounts.js
       filters: browserify
       contents:
         - h:static/scripts/config/accounts.coffee
+    - output: scripts/config.via.js
```

Comment:
> I would just add this content to the bundle above.

I tried to, but I had trouble figuring our the correct browserify syntax. Here is the problem: browserify doesn't take a list of input files; it takes one file, and optionally a list of dependencies. It didn't feel right to add the via config file as a dependency to the accounts config file, so I didn't want to do that.

I guess there must be a better way; please show it. :)

> Or update `make clean` at your preference.

Fortunately, `make clean` already has this line:

```
rm -f h/static/scripts/config.*js
```

... which covers all these files. So I think we are good.